banzai/utils/image_utils.py

```diff
@@ -106,36 +110,44 @@ def save_images(pipeline_context, images, master_calibration=False):
     return output_files
 
 
-def get_bpm(image, pipeline_context):
+def load_bpm(image, pipeline_context):
     bpm_filename = dbs.get_bpm(image.telescope_id, image.ccdsum,
                                db_address=pipeline_context.db_address)
-    if bpm_filename is None:
-        bpm = None
-        image.header['L1IDMASK'] = ('', 'Id. of mask file used')
+    if pipeline_context.bpm_required and bpm_filename is None:
+        raise MissingBPMError('No Bad Pixel Mask file exists for this image.')
+    elif bpm_filename is None:
+        load_default_bpm(image)
+    else:
+        load_bpm_file(bpm_filename, image)
+
+
+def load_default_bpm(image):
```

Comment:
Maybe call this "null" instead of "default"?

Reply:
I agree